src/Result/Webhook.php

```diff
@@ -6,4 +6,39 @@
 
 class Webhook extends AbstractResult
 {
+    public function getId(): string
+    {
+        $data = $this->getData();
+        return $data['id'];
+    }
+
+    public function isEnabled(): bool
+    {
+        $data = $this->getData();
+        return $data['enabled'];
+    }
+
+    public function hasAutomaticRedelivery(): bool
+    {
+        $data = $this->getData();
+        return $data['automaticRedelivery'];
+    }
+
+    public function getUrl(): string
+    {
+        $data = $this->getData();
+        return $data['url'];
+    }
+
+    public function getAuthorizedEvents(): WebhookAuthorizedEventList
+    {
+        $data = $this->getData();
+        return new WebhookAuthorizedEventList($data['authorizedEvents']);
+    }
+
+    public function getSecret(): string
```

Comment:
Not sure if the secret is returned here, can't see it in the docs https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Webhokks_GetWebhook but it is listed on the createWebhook, where it makes sense as you can set it yourself. But seems not to be available here?

Reply:
You're right. I did add it because of the create webhook but yeah it doesn't really make sense to grab it again via API if it's set by the user. Removing.